### ISSUE TITLE
Uwp crash Unspecified error

### ISSUE THREAD

**Issue by eliasnawfal** [alexrainman/CarouselView#177]
I m using this component with Bindable ItemSource on Android, iOS and Uwp.
The ItemSource is an ObservableCollection. On Android and iOS everything works fine but on Uwp if I swipe down the control center then swipe up. The data disappears and app crashes with an Unspecified Error. The app only crashes in the page where I use the component.

I would appreciate any help
Thank you for your good work

TL;DR: Binding breaks on Uwp if I swipe control center down

**Comment by alexrainman** [comment]
I did a change in latest version that has broken UWP. Please wait for next release.
